docs/src/main/sphinx/connector/teradata.md

```diff
@@ -0,0 +1,214 @@
+# Teradata connector
+
+```{raw} html
+<img src="../_static/img/teradata.png" class="connector-logo">
+```
+
+The Teradata connector allows querying and creating tables in an external
+[Teradata](https://www.teradata.com/) database. This can be used to join 
+data between different systems like Teradata and Hive, or between different Teradata instances.
+
+## Requirements
+
+To connect to Teradata, you need:
+
+- Teradata Database
+- Network access from the Trino coordinator and workers to Teradata. Port 
+  1025 is the default port
+
+## Configuration
+
+To configure the Teradata connector, create a catalog properties file in
+`etc/catalog` named, for example, `example.properties`, to mount the Teradata
+connector as the `teradata` catalog. Create the file with the following 
+contents, replacing the connection properties as appropriate for your setup:
+
+```properties
+connector.name=teradata
+connection-url=jdbc:teradata://example.teradata.com/CHARSET=UTF8,TMODE=ANSI,LOGMECH=TD2
+connection-user=***
+connection-password=***
+```
+
+The `connection-url` defines the connection information and parameters to pass
+to the Teradata JDBC driver. The supported parameters for the URL are 
+available in the 
+[Teradata JDBC documentation](https://teradata-docs.s3.amazonaws.com/doc/connectivity/jdbc/reference/current/jdbcug_chapter_2.html#BABJIHBJ).
+For example, the following `connection-url` configures character encoding, 
+transaction mode, and authentication.
+
+```properties
+connection-url=jdbc:teradata://example.teradata.com/CHARSET=UTF8,TMODE=ANSI,LOGMECH=TD2
+```
+
+The `connection-user` and `connection-password` are typically required and 
+determine the user credentials for the connection, often a service user.
+
+### Connection security
+
+If you have TLS configured with a globally-trusted certificate installed on 
+your data source, you can enable TLS between your cluster and the data 
+source by appending parameters to the JDBC connection string set in the 
+connection-url catalog configuration property.
+
+For example, to specify `SSLMODE`:
+
+```properties
+connection-url=jdbc:teradata://example.teradata.com/SSLMODE=REQUIRED
+```
+
+For more information on TLS configuration options, see the
+Teradata [JDBC documentation](https://teradata-docs.s3.amazonaws.com/doc/connectivity/jdbc/reference/current/jdbcug_chapter_2.html#URL_SSLMODE).
+
+```{include} jdbc-authentication.fragment
+```
+
+### Multiple Teradata databases
+
+You can have as many catalogs as you need, so if you have additional Teradata
```

Comment:
```suggestion
You can configure as many catalogs as you need, so if you have additional Teradata
```

Reply:
Revised `Multiple Teradata databases` section to align with Postgres and SQL Server connector content.